styles.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] I suggest you add a CSS style as required ⬇️ 
![image](https://github.com/Drew246/firstMicroProject/assets/68823331/b79df615-c2bd-4937-9155-a799de6ee3dd)